packages/frameworks-solid-start/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@auth/solid-start",
+  "description": "Authentication for SolidStart.",
+  "version": "0.0.1",
+  "type": "module",
+  "files": [
+    "handler.*",
+    "session.*",
+    "utils.*",
+    "src"
+  ],
+  "exports": {
+    "./handler": {
+      "types": "./handler.d.ts",
+      "import": "./handler.js"
+    },
+    "./session": {
+      "types": "./session.d.ts",
+      "import": "./session.js"
+    },
+    "./utils": {
+      "types": "./utils.d.ts",
+      "import": "./utils.js"
+    },
```

Comment:
Let's rename the following to match SvelteKit/upcoming auth packages:

- `@auth/solid-start/handler` -> `@auth/solid-start`
- `@auth/solid-start/session` -> `@auth/solid-start`
- `@auth/solid-start/utils` -> `@auth/solid-start/client`

Reply:
done